content/webapp/services/prismic/types/contributors.ts

```diff
@@ -43,7 +42,7 @@ export type Organisation = PrismicDocument<
     description: RichTextField;
     image: Image;
     sameAs: GroupField<{
-      link: LinkField;
+      link: KeyTextField;
```

Comment:
See https://github.com/wellcomecollection/wellcomecollection.org/blob/main/prismic-model/src/people.ts#L19-L22 and https://github.com/wellcomecollection/wellcomecollection.org/blob/main/prismic-model/src/organisations.ts#L19-L22 – both Person and Organisation use a text field as a link.